server/utils/assessmentUtils.ts

```diff
@@ -25,35 +25,49 @@ export const formatAssessmentData = (
     .filter(areaConfig => areaConfig.crimNeedsKey in crimNeeds && crimNeeds[areaConfig.crimNeedsKey])
     .map(areaConfig => {
       let score
-      const isLinkedtoRoSH = crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}LinkedToHarm`] === 'YES'
-      const isLinkedtoReoffending =
-        crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}LinkedToReoffending`] === 'YES'
-      const isLinkedtoStrengthsOrProtectiveFactors =
-        crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}Strengths`] === 'YES'
+
+      const linkedtoHarm = crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}LinkedToHarm`].toLowerCase()
+      const linkedtoReoffending =
+        crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}LinkedToReoffending`].toLowerCase()
+      const linkedtoStrengthsOrProtectiveFactors =
+        crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}Strengths`].toLowerCase()
+
       let subData: SubAreaData
       let overallScore
       const motivationToMakeChanges = motivationText(
         assessment.sanAssessmentData[`${areaConfig.assessmentKey}_changes`]?.value,
       )
 
-      const linkedToRoSHYesNo = isLinkedtoRoSH ? 'yes' : 'no'
-      const linkedToReoffendingYesNo = isLinkedtoReoffending ? 'yes' : 'no'
-      const linkedtoStrengthsOrProtectiveFactorsYesNo = isLinkedtoStrengthsOrProtectiveFactors ? 'yes' : 'no'
-
       const riskOfSeriousHarmDetails =
-        assessment.sanAssessmentData[
-          `${areaConfig.assessmentKey}_practitioner_analysis_risk_of_serious_harm_${linkedToRoSHYesNo}_details`
-        ]?.value
+        linkedtoHarm !== 'null'
+          ? assessment.sanAssessmentData[
+              `${areaConfig.assessmentKey}_practitioner_analysis_risk_of_serious_harm_${linkedtoHarm}_details`
```

Comment:
presumably this kind of null check is what we should be doing up on lines 29-32 now that we're not generating the boolean values